### PR TITLE
Cfy 5210 move restservice packaging to omnibus

### PR DESCRIPTION
**Pull request by gilzellner**

### CODE REVIEW

File: packaging/restservice/omnibus/.gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.DS_Store
+*.gem
```
Comment (gilzellner) [inline packaging/restservice/omnibus/.gitignore]: remove

File: packaging/restservice/omnibus/.kitchen.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
+driver:
```
Comment (gilzellner) [inline packaging/restservice/omnibus/.kitchen.yml]: remove

File: packaging/restservice/omnibus/.travis.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (gilzellner) [inline packaging/restservice/omnibus/.travis.yml]: remove

File: packaging/restservice/omnibus/Berksfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+source 'https://supermarket.chef.io'
```
Comment (gilzellner) [inline packaging/restservice/omnibus/Berksfile]: remove

File: packaging/restservice/omnibus/Gemfile [hunk 5]
```diff
@@ -0,0 +1,24 @@
+source 'https://rubygems.org'
```
Comment (gilzellner) [inline packaging/restservice/omnibus/Gemfile]: remove

File: packaging/restservice/omnibus/README.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
+cloudify Omnibus project
```
Comment (gilzellner) [inline packaging/restservice/omnibus/README.md]: rewrite

File: packaging/restservice/omnibus/package-scripts/rest-service/prerm [hunk 7]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Perform necessary setup steps
+# prior to installing package.
+#
+
+PROGNAME=$(basename $0)
+
+function error_exit
```
Comment (gilzellner) [inline packaging/restservice/omnibus/package-scripts/rest-service/prerm]: do we need this ?

File: packaging/restservice/omnibus/rest-service/recipes/default.rb [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  execute 'build project' do
+    cwd node['omnibus']['build_dir']
+    command <<-EOF.gsub(/\s+/, ' ').strip!
+      sudo -i -u #{node['omnibus']['build_user']} \
+      bash -l -c 'source load-omnibus-toolchain.sh; \
+      cd #{node['omnibus']['build_dir']}; \
+      bundle exec omnibus build #{node['omnibus']['build_project']} > #{logdir}/#{node['hostname']}-build-project.log 2>&1'
+      EOF
+  end
+when 'windows'
```
Comment (gilzellner) [inline packaging/restservice/omnibus/rest-service/recipes/default.rb]: remove windows stuff

File: packaging/restservice/omnibus/rest-service/recipes/install.rb [hunk 9]
```diff
@@ -0,0 +1,23 @@
+
+# reload ohai data
+ohai 'reload' do
+  action :reload
+end
+
+# should only be one of each rpm/deb
+pkg_path = Dir.glob("#{node['etc']['passwd']['vagrant']['dir']}/cloudify/pkg/cloudify*rpm")[0] if node['platform_family'] == 'rhel'
+pkg_path = Dir.glob("#{node['etc']['passwd']['vagrant']['dir']}/cloudify/pkg/cloudify*deb")[0] if node['platform_family'] == 'debian'
```
Comment (gilzellner) [inline packaging/restservice/omnibus/rest-service/recipes/install.rb]: remove debian stuff

File: packaging/restservice/omnibus/rest-service/recipes/install.rb [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+pkg_path = Dir.glob("#{ENV['HOME']}/pkg/cloudify*msi")[0] if node['platform_family'] == 'windows'
```
Comment (gilzellner) [inline packaging/restservice/omnibus/rest-service/recipes/install.rb]: remove windows stuff

File: packaging/restservice/omnibus/test/integration/cloudify/serverspec/localhost/cloudify_spec.rb [hunk 11]
```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+if ['centos', 'redhat'].include?(os[:family])
```
Comment (gilzellner) [inline packaging/restservice/omnibus/test/integration/cloudify/serverspec/localhost/cloudify_spec.rb]: remove commented out code

**Comment by geokala** [comment]
Closed: As this PR is more than six months old without having been merged. Please reopen (and rebase if necessary) if it is still relevant.

**Pull request closed without merging**